### PR TITLE
tests,po: sync translations from LP and add regression test for LP:1723974

### PR DESCRIPTION
**Pull request by mvo5**
Fwiw, ths will fail currently because we actually crash for some locales - this will get fixed with #4050.

### CODE REVIEW

**Comment by zyga** [comment]
Holly molly.

File: tests/main/i18n/task.yaml [hunk 1]
```diff
@@ -13,4 +13,11 @@ execute: |
     fi
 
     echo "Ensure that i18n works"
-    LANG=de_DE.UTF-8 snap changes everything | MATCH "Ja, ja, allerdings."
+    LANG=de snap changes everything | MATCH "Ja, ja, allerdings."
+
+    echo "Basic smoke test to ensure no locale causes crashes"
+    for p in /usr/share/locale/*; do
+        export LANG=$(basename $p)
```
Comment (zyga) [inline tests/main/i18n/task.yaml]: I think you may need something like `language-support-XX` for this to work

File: tests/main/i18n/task.yaml [hunk 2]
```diff
@@ -13,4 +13,11 @@ execute: |
...
+        snap >/dev/null
```
Comment (zyga) [inline tests/main/i18n/task.yaml]: Ideally we'd run a small smoke test here (snap install, snap refresh, snap remove). I worry about things that break when localized.

File: tests/main/i18n/task.yaml (hunk 1 above)
Comment (mvo5) [reply tests/main/i18n/task.yaml]: While true in the general case, we use our own i18n native go code which bypasses the need for language-support-XX .

**Pull request merged**